### ISSUE TITLE
Can't log into settings

### ISSUE THREAD

**Issue by gdtms247** [akasma74/Hass-Custom-Alarm#111]
**The component:**
- Home Assistant: 0.112.4
- BWAlarm (ak74 edition): v1.12.6
- Hassio VM

Setup is only a few days old, on initial setup I logged into the alarm, updated password and choose the devices to trigger alarm etc. Now that I have it working I want to remove the test device and noticed my password no longer works. Tried the default and it doesn't work either. I've reinstalled, manually deleted and re-added the entire custom components folder for bwalarm and no change. I can see the following in my alarm_control_panel.py:

    #---------------------------PANEL RELATED---------------------------
    vol.Optional(CONF_ADMIN_PASSWORD, default='HG28!!&dn'):       cv.string,     # Admin panel password

However on the login page this password doesn't bring up the settings menu. Anything else to try?

**Comment by akasma74** [comment]
Sounds like you used HACS..
Anyway, do you know where your `bwalarm.yaml` resides?
It should be in `<HA config folder>/resources/bwalarm` and if you open it, the password is there as `admin_password`.
If it's there and yet you cannot get into Settings, please update the issue posting your `bwalarm.yaml` here.

**Comment by gdtms247** [comment]
Yep that was it, I don't know how that got changed but it was set to my alarm panel arm/disarm code. Thanks!!

**Comment by akasma74** [comment]
Cool.
Could you change it as you did it before and keep an eye on it for a while?
If it doesn't get changed and everythig works as expected, then there's nothing to fix and you can close the issue.
Just take your time ;)

**Comment by gdtms247** [comment]
Will do! I'll admit I probably did it to myself :) but i'll keep my eye on it next week. thanks!

**Comment by akasma74** [comment]
Is the issue resolved or you still experience it?
